src/main/scala/ru/gaptrakhmanov/telegram/helper/util/UrlUtils.scala

```diff
@@ -0,0 +1,23 @@
+package ru.gaptrakhmanov.telegram.helper.util
+
+import cats.effect.{Resource, Sync}
+
+import scala.io.{BufferedSource, Source}
+
+object UrlUtils {
```

Comment:
I will make all the def private and leave only the `getDataFromUrl` public in this way we keep in the implementation internally and we don't leak a connection if someone only uses `acquire`.